cli/content-version.js

```diff
@@ -0,0 +1,53 @@
+const fs = require("fs");
+const path = require("path");
+const { Repository } = require("nodegit");
+
+const PROJECT_ROOT = path.join(__dirname, "..");
+const SHA_PATH = path.join(PROJECT_ROOT, "content.gitsha");
+const CONTENT_NAME = "stumptown";
+
```

Comment:
In `ssr/index.js` we do this:
```javascript
const STUMPTOWN_CONTENT_ROOT =
  process.env.STUMPTOWN_CONTENT_ROOT || path.join(PROJECT_ROOT, "stumptown");
```
then you could replace `path.join(PROJECT_ROOT, CONTENT_NAME)` with `STUMPTOWN_CONTENT_ROOT` on line 11.